### PR TITLE
Update LinkedIn link

### LINKED ISSUE #400: Change the LinkedIn link
**Is your feature request related to a problem? Please describe.**
Currently, I have changed my linkedin account and in the readme.md file its the wrong link

**Describe the solution you'd like**
Change the LinkedIn link

New link :- https://www.linkedin.com/in/YoshithaRathnayake

**Are you work on this?**
No

### PR DESCRIPTION
**Pull request by Achal-00**
## Purpose

The purpose of this Pull Request is to fix #400

## Goals
update the LinkedIn link in readme

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by YoshithaRathnayake — APPROVED** [review]
Good job @Achal-00

**Review by YoshithaRathnayake — APPROVED** [review]
Thank You! @Achal-00

**Review by YoshithaRathnayake — APPROVED** [review]

**Comment by YoshithaRathnayake** [comment]
Seem to be the commits are not signed. @Achal-00 So I can't merge it
Can you find a solution or if not you can create a new pr

File: README.md [hunk 1]
```diff
@@ -269,13 +269,14 @@ const info = {
...
...mg src="https://img.icons8.com/color/48/000000/whatsapp--v1.png" alt="WhatsApp Icon" title="WhatsApp"/></a> <a href="https://t.me/joinchat/hP-WbTXCf185YTg1"><img src="https://img.icons8.com/fluency/48/telegram-app.png" alt="Telegram Icon" title="Telegram"/></a> <a href="https://github.com/YoshithaRathnayake"><img src="https://img.icons8.com/fluency/48/000000/github.png" alt="GitHub Icon" title="GitHub"/></a> <a href="https://gitlab.com/YoshithaRathnayake"><img src="https://img.icons8.com/color/48/000000/gitlab.png" alt="GitLab Icon" title="GitLab"/></a> <a href="https://twitter.com/Yoshitha2008"><img src="https://img.icons8.com/color/48/000000/twitter-circled--v1.png" alt="Twitter Icon" title="Twitter"/></a> <a href="https://www.instagram.com/YoshithaRathnayake"><img src="https://img.icons8.com/color/48/000000/instagram-new.png" alt="Instagram Icon" title="Instagram"/></a> <a href="https://www.linkedin.com/in/YoshithaRathnayake"><img src="https://img.icons8.com/fluency/48/linkedin.png" alt="LinkedIn Icon" title="LinkedIn"/></a> <a href="https://yoshitharathnayake.medium.com"><img src="https://img.icons8.com/ios-filled/48/ffffff/medium--v2.png" alt="Medium Icon" title="Medium"/></a>
```
Comment (YoshithaRathnayake) [inline README.md:272]: Can you remove this line? @Achal-00 
```suggestion

```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Achal-00) [reply README.md]: @YoshithaRathnayake I have removed those lines.

**Review by YoshithaRathnayake — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (YoshithaRathnayake) [reply README.md]: Ok great! @Achal-00

**Comment by YoshithaRathnayake** [comment]
Don't forget to follow me in github and other social media @Achal-00

**Pull request merged**